doc/plugins/unittest.rst

```diff
@@ -0,0 +1,69 @@
+###############
+Spyder Unittest
+###############
+
+**Spyder-unittest** is a plugin that integrates popular unit test frameworks with Spyder, allowing you to run test suites and view the results in the IDE.
+
+.. image:: /images/unittest/unittest-run.png
+   :alt: Spyder Unittest in Spyder
+
+Using unittest inside Spyder allows you to take advantage of their web interface alongside Spyder’s powerful features such as the Variable explorer, console and debugger.
+
+===================
+Installing Unittest
+===================
+
+.. code-block:: bash
+
+   conda install -c spyder-ide spyder-unittest
+
+.. important::
+
+   Please make sure you have Python's pytest package installed if you are going to use that program as well.
+
+.. code-block:: bash
+
+   conda install pytest
+
+Restart Spyder in order to be able to use the plugin.
+
+==============
+Using Unittest
+==============
+
+When Unittest is installed, it will be available under the menu item :menuselection:`View --> Panes --> Unit testing`.
+
+.. image:: /images/unittest/unittest-view-panes.png
+   :alt: Spyder showing view panes Unittest
+
+You will see it then as a tab in the bottom of the top right area in the same location as the Variable explorer, Help, and Plots views.
+
+Press on the  :guilabel:`Run Green Arrow` to run the testing.
+
+.. image:: /images/unittest/unittest-run-button.png
+   :alt: Spyder showing Unittest run button
+
+You can get an expanded view of the testing messages by clicking on the expand option.
+
+.. image:: /images/unittest/unittest-expanded-message-view.png
+   :alt: Spyder showing Unittest expanded message view
+
+Press on the  :guilabel:`Hamburger Menu Icon` to view different unittest options.
```

Comment:
Before this one I think it could be worth adding the "Advanced options" since these options are not that common.

Reply:
Agreed.

Reply:
Added information on the configuration view, dependencies view, and the output viewer.